config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  get '/events', to: 'events#index'
+  root to: 'events#index'
+  get '/events', to: 'events#events'
 end
```

Comment:
The route '/events' should be named more descriptively. Consider using a more meaningful name like '/events/:id' to show a specific event or '/events/new' to create a new event.